README.md

```diff
@@ -1,126 +1,120 @@
 # Test Network Function ![build](https://github.com/test-network-function/test-network-function/actions/workflows/merge.yaml/badge.svg) [![Go Report Card](https://goreportcard.com/badge/github.com/test-network-function/test-network-function)](https://goreportcard.com/report/github.com/test-network-function/test-network-function)
 
-This repository contains a set of network function test cases and the framework to build more.  It also generates reports
-(claim.json) on the result of a test run.
 
-Please consult [CATALOG.md](./CATALOG.md) for a catalog of the included test cases and test case building blocks.
+This repository contains a set of Cloud-Native Network Functions (CNFs) test cases and the framework to build more. The tests and framework are intended 
+to test the interaction of CNFs with OpenShift Container Platform.  It also generates a report 
+(claim.json) after running tests.
 
-The tests and framework are intended to test the interaction of Cloud-Native Network Functions (CNFs) with OpenShift
-Container Platform.
+Please consult [CATALOG.md](./CATALOG.md) for a catalog of the included test cases and test case building blocks.
 
 The suite is provided here in part so that CNF Developers can use the suite to test their CNFs readiness for
 certification.  Please see "CNF Developers" below for more information.
 
-## Container
+## Overview 
+ ![overview](docs/images/overview.svg)
 
-### Running
+In the diagram above:
+- the `CNF under test` is the CNF to be certified. The certification suite identifies the pods belonging to the CNF via a label, see [Pod Roles](#pod_roles)
+- the `Certification container/exec` is the certification testsuite running on the platform or in a container. The executable verifies the CNF under test configuration and its interactions with openshift 
+- the `Partner pod` is a pod designated to be the remote end of test requiring 2 partner, the other partner being the CNF under test. For instance ping and iperf (TODO) are example of test requiring a partner. The partner pod can be any pod with the proper tools (ping, iperf, ...) in the same namespace as the CNF under test. The certification suite will remotely connect to it to run test commands.
```

Comment:
``the `Partner pod` is a pod designated to be the remote end of test requiring 2 partner`` can be reformulated for something easier ?

Reply:
should the ``(TODO)``  should be removed. or it's a place holder for something ?

Reply:
ok, will improve and remove the TODO, thanks